src/pyshark/capture/pipe_ring_capture.py

```diff
@@ -0,0 +1,87 @@
+from pyshark.capture.pipe_capture import PipeCapture
+from pyshark.capture.capture import TSharkCrashException, TSharkVersionException
+from pyshark.tshark.tshark import tshark_supports_json
+import asyncio
+import subprocess
+import tempfile
+
+
+class DisplayFilterNotAllowedException(Exception):
```

Comment:
Due to how RingCaptures work, Tshark will not allow a display filter in this class.